packages/blitz/src/utils/index.ts

```diff
@@ -143,3 +143,11 @@ export function interopDefault(mod: any) {
 export function truncateString(str: string, maxLength: number): string {
   return str.length > maxLength ? str.substring(0, maxLength - 3) + "..." : str
 }
+
+export function isNotInUserTestEnvironment() {
+  if (process.env.VITE_BLITZ_TEST_ENVIRONMENT) {
+    return true
+  }
+
+  return process.env.JEST_WORKER_ID === undefined && process.env.VITEST_WORKER_ID === undefined
+}
```

Comment:
Hey @jhonnymichel, I was just taking a look at this, and haven't tested it yet. 

Logically for a function named `isNotInUserTestEnvironment` it should return `true` in normal execution and `false` in tests. 

So shouldn't after checking the `env` variable `VITE_BLITZ_TEST_ENVIRONMENT` that you are setting for the tests, shouldn't the function return `false` because it is in the test env. 

If it's a misconception on my part, or if I missing something, do let me know.

```suggestion
export function isNotInUserTestEnvironment() {
  if (process.env.VITE_BLITZ_TEST_ENVIRONMENT) {
    return false
  }

  return process.env.JEST_WORKER_ID === undefined && process.env.VITEST_WORKER_ID === undefined
}
```

Reply:
The idea is to return `false` if we're in a 'blitz user' (a next app using the blitz toolkit) test environment.  it will return `true` while running in the blitz repo (this repo) tests.

That's why the function is called `isNotInUserTestEnvironment` instead of just `isNotInTestEnvironment`. the name is also not mine, I just moved the function around, but it already existed with the same strategy -- although it had a bug, according to my judgement.

Previously this check (wanting to know if we're running inside a blitz app, not inside the blitz repo) existed, but it was simply `if (process.env.JEST_WORKER_ID === undefined)`, because the expectation was that the blitz repo tests ran in vitest, and user (a next app using the blitz toolkit) tests ran in jest.

This not being the case anymore, we need a flag to differentiate the two cases.

Also, this flag strategy already existed in the codebase at some point, probably when the blitz repo tests also ran with Jest. and now I'm kinda recovering it and using it when needed.

Reply:
Hey @jhonnymichel, thanks a lot for the clarification!